src/stage1/compiler_utils.jl

```diff
@@ -7,7 +7,7 @@ function Base.push!(cfg::CFG, bb::BasicBlock)
     push!(cfg.index, bb.stmts.start)
 end
 
-if VERSION <= v"1.11.0-DEV.116"
+if VERSION < v"1.11.0-DEV.258"
```

Comment:
I am not sure where 116 came from but it wasn't working on the Cedar blessed just version which is greeater than 116.
SO I checked blame and found 
https://github.com/JuliaLang/julia/commit/517ad06a8e9f1f8ec3b80022fef5f32a717d6dd3
which is 258

Reply:
this came from me making up a number :).